src/averageAges.js

```diff
@@ -55,7 +93,26 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  let count = 0;
+
+  const childAndMoms = people.reduce((perv, child) => {
```

Comment:
Use a more descriptive variable name instead of `childAndMoms`. Consider using `totalAgeDifference` for better readability.